docs/pages/developer-guide.md

```diff
@@ -1308,6 +1308,62 @@ $ py.test frameworks/helloworld/
 
 The most basic set of features present in the YAML representation of the `ServiceSpec` are [presented above](#service-spec). The remaining features are introduced below.
 
+### Pods
+
+You may specify the number of pod instances to be run for every pod. As a safety measure, after initial install users can increase but not decrease this value. If you wish to allow scale-in of your pods, you must specify `allow-decommission: true` for each applicable pod, like this:
```

Comment:
"after initial install, users can increase but not decrease this value."

remove ", like this"

Reply:
Done